community/contributing/bisect_workflow.rst

```diff
@@ -0,0 +1,136 @@
+it can also be used to find other kinds of unexpected changes such as
+performance regressions.
+
+Using official builds to speed up bisecting
+-------------------------------------------
+
+Before using Git's ``bisect`` command, we strongly recommend trying to reproduce
+the bug with an older (or newer) official release. This greatly reduces the
+range of commits that potentially need to be built from source and tested.
+You can find binaries of official releases, as well as alphas, betas,
+and release candidates `here <https://downloads.tuxfamily.org/godotengine/>`__.
+
+For example, if you've reported a bug against Godot 3.2 RC4, you should first
+try to reproduce the bug in Godot 3.1 (not a patch release, see below for the
+reason). If the bug doesn't occur there, try to reproduce it in Godot 3.2 alpha
+1. If the bug doesn't occur there either, the regression appeared between Godot
+3.1 and 3.2 alpha 1. Otherwise, it appeared in a newer pre-release build; you
+should now test newer alphas, betas and RC builds.
+
+.. warning::
+
+    For bisecting regressions, don't use patch releases such as Godot 3.1.2.
+    Instead, use the minor version's first release like Godot 3.1. This is
+    because patch releases are built from a separate *stable branch*. This kind
+    of branch doesn't follow the rest of Godot's development, which is done in
+    the ``master`` branch.
+
+The Git bisect command
+----------------------
+
+If you've found a build that didn't exhibit the bug in the above testing
+process, you can now start bisecting the regression. The Git version control
+system offers a built-in command for this: ``git bisect``. This makes the
+process semi-automated as you only have to build the engine, run it and try to
+reproduce the bug.
+
+.. note::
+
+    Before bisecting a regression, you need to set up a build environment to
+    compile Godot from source. To do so, read the
+    :ref:`Compiling <toc-devel-compiling>` page for your target platform.
+    (Compiling Godot from source doesn't require C++ programming knowledge.)
+
+To start bisecting, you must first determine the commit hashes (identifiers) of
+the "bad" and "good" build. "bad" refers to the build that exhibits the bug,
+whereas "good" refers to the version that doesn't exhibit the bug. If you're
+using a pre-release build as the "good" or "bad" build, browse the `download
+mirror <https://downloads.tuxfamily.org/godotengine/>`__, go to the folder that
+contains the pre-release you downloaded and look for the ``README.txt`` file.
+The commit hash is written inside that file.
+
+If you're using a stable release as the "good" or "bad" build, use one of the
+following commit hashes depending on the version:
+
+.. code-block:: none
+
+    3.1: 320f49f204cfbf9b480fe62aaa7718afb74920a5
+    3.0: 9bd402698cfa299b79b9144f8d7744c308a4e085
+
+To refer to the latest state of the master branch, you can use ``HEAD``
+instead of a commit hash.
```

Comment:
Actually I think using `master` would be better. `HEAD` is the currently checked out commit, which may not be the `master` one if they're on another branch, or if they already checked out an older commit to try to confirm a `good` before starting a bisect.